chain/network/src/peer.rs

```diff
@@ -68,6 +71,10 @@ pub const EPOCH_SYNC_REQUEST_TIMEOUT_MS: u64 = 1_000;
 /// How frequently a Epoch Sync response can be sent to a particular peer
 // TODO #3488 set 60_000
 pub const EPOCH_SYNC_PEER_TIMEOUT_MS: u64 = 10;
+/// Limit cache size of 1000 messages
+pub const ROUTED_MESSAGE_CACHE_SIZE: usize = 1000;
+/// Duplicated messages will be dropped if routed through the same peer multiple times.
+pub const DROP_DUPLICATED_MESSAGES_PERIOD: Duration = Duration::from_millis(100);
```

Comment:
Let's change this to 50ms to err on the side of caution

Reply:
Done